src/index.scss

```diff
@@ -7,11 +7,17 @@
   font-style: normal;
 }
 
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/AvenirNextLTPro-Regular.otf");
```

Comment:
import the font as `url("/fonts/AvenirNextLTPro-Regular.otf")`

Wrong imports works in the development because of vite but won't work in production. So check the imports. As mentioned many times vite serves the public folder as root "/" so that "/" is necessary